bridges/testing/environments/rococo-westend/bridges_rococo_westend.sh

```diff
@@ -250,6 +250,21 @@ case "$1" in
           "$GLOBAL_CONSENSUS_WESTEND_SOVEREIGN_ACCOUNT" \
           10000000000 \
           true
+      # create foreign asset pool
+      force_create_pool \
```

Comment:
very nit, both `force_create_pool` / `force_add_liquidity` are not really `force`, please remove prefix `force_` :)